sdk/keyvault/keyvault-certificates/src/core/models/index.ts

```diff
@@ -2299,20 +2299,20 @@ export interface KeyVaultClientUpdateSasDefinitionOptionalParams
 }
 
 /**
- * Defines values for JsonWebKeyType.
+ * Defines values for KeyType.
  * Possible values include: 'EC', 'EC-HSM', 'RSA', 'RSA-HSM', 'oct'
  * @readonly
  * @enum {string}
  */
-export type JsonWebKeyType = "EC" | "EC-HSM" | "RSA" | "RSA-HSM" | "oct";
+export type KeyType = "EC" | "EC-HSM" | "RSA" | "RSA-HSM" | "oct";
 
 /**
- * Defines values for JsonWebKeyCurveName.
+ * Defines values for KeyCurveName.
  * Possible values include: 'P-256', 'P-384', 'P-521', 'P-256K'
  * @readonly
  * @enum {string}
  */
-export type JsonWebKeyCurveName = "P-256" | "P-384" | "P-521" | "P-256K";
+export type KeyCurveName = "P-256" | "P-384" | "P-521" | "P-256K";
```

Comment:
This is autogenerated code based off the swagger. Would it be possible to do the rename where we import / re-export instead?

Reply:
Updated this PR to just alias these interfaces for now, but made a PR to fix this in swagger: https://github.com/Azure/azure-rest-api-specs/pull/7651